pip/req/req_set.py

```diff
@@ -489,8 +489,13 @@ def _prepare_file(self,
                     'req_to_install.satisfied_by is set to %r'
                     % (req_to_install.satisfied_by,))
                 logger.info(
-                    'Requirement %s: %s', skip_reason,
-                    req_to_install)
+                    'Requirement %s: %s %s', skip_reason,
+                    req_to_install,
+                    "(" + str(req_to_install.satisfied_by.version) + ")"
+                        if skip_reason in
+                            ('already satisfied',
+                            'already up-to-date')
+                        else '')
```

Comment:
We always have a `req_to_install.satisfied_by` and a `skip_reason` here.
Let's keep it simple and use:

    logger.info('Requirement %s: %s (%s)', skip_reason, req_to_install, req_to_install.satisfied_by.version)

or am I missing something ?